python/ray/tune/trial_runner.py

```diff
@@ -379,7 +380,11 @@ def _memory_debug_string(self):
             return "Memory usage on this node: {}/{} GB{}".format(
                 round(used_gb, 1), round(total_gb, 1), warn)
         except ImportError:
-            return ("Unknown memory usage. Please run `pip install psutil` "
+            global _MEMORY_USAGE_WARNED
+            if not _MEMORY_USAGE_WARNED:
+                _MEMORY_USAGE_WARNED = True
```

Comment:
Please don't do this. The user can easily miss this warning otherwise since it's relatively unobtrusive.